git_machete/cli.py

```diff
@@ -756,7 +756,7 @@ def strip_remote_name(remote_branch: RemoteBranchShortName) -> LocalBranchShortN
                 if 'branch' in parsed_cli:
                     raise MacheteException(
                         '`show current` with a <branch> argument does not make sense')
-                print(branch)
+                print(bold(branch))
```

Comment:
Whoops here it doesn't make sense, `show` is a porcelain command anyway, more for the use in scripts than by a user directly. Revert to `print(branch)`